tools/blupdater/blupdater.ld

```diff
@@ -20,7 +20,8 @@ SECTIONS
         KEEP(*(.header));
     } >rom AT >rom
 }
-INCLUDE libopencm3_stm32f2.ld
+
+INCLUDE cortex-m-generic.ld
```

Comment:
libopencm3 no longer provides stm32f2-specific linker scripts, so we'll have to go with the generic one (which is good enough)